block/stream/output/schedule_service.proto

```diff
@@ -64,6 +64,17 @@ import "basic_types.proto";
  * the original transaction.
  */
 message CreateScheduleOutput {
+    /**
+     * A schedule identifier.
+     * <p>
+     * If the status of the transaction is `SUCCESS`, this value SHALL be
+     * set to the identifier of the schedule that was created. If the
+     * transaction status is `IDENTICAL_SCHEDULE_ALREADY_CREATED`, this
+     * value SHALL be set to the identifier of the existing schedule that
+     * is identical to the one that was attempted to be created. For any
+     * other status, this value SHALL NOT be set.
+     */
+    proto.ScheduleID schedule_id = 1;
```

Comment:
I do not believe this is needed.  The new `Schedule` that is created will be present as a StateChange and will contain the ID.

Reply:
In the case of an `IDENTICAL_SCHEDULE_ALREADY_CREATED`, the existing schedule's id is set in the receipt; but that existing schedule will not be changed in the transaction.

Reply:
That makes sense.
Worth noting that it is very important to avoid sending any data unless strictly necessary in BlockStream.  Every byte has a permanent ongoing cost to store (there is no purge or delete for block stream), so the less we write the less we end up committing to spend forever.

If we can avoid setting this (e.g. in every case _except_ duplicate), we should.

Reply:
Ah yes that is very sensible, will make sure it works that way. 💯